model-optimizer/mo/utils/cli_parser.py

```diff
@@ -1127,6 +1130,85 @@ def get_absolute_path(path_to_file: str) -> str:
     return file_path
 
 
+def isfloat(value):
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def detect_value_type(value: str):
```

Comment:
The function does not detect value type. It actually converts from string to an appropriate data type. Please, update the name.

Reply:
Done.